modules/transactionpool/accept.go

```diff
@@ -284,7 +314,8 @@ func (tp *TransactionPool) acceptTransactionSet(ts []types.Transaction, txnFn fu
 		}
 	}
 	if len(conflicts) > 0 {
-		return tp.handleConflicts(ts, conflicts, txnFn)
+		err := tp.handleConflicts(ts, conflicts, txnFn)
+		return err
```

Comment:
I personally prefer the return in the same line. Do we have a convention for that?